Joesdev/dog-messenger
# FASP-15 Provide 'Distance' as a part of the data for Results Page

## Whats this PR do?
- Adds distance on results.blade, which is set by the user at the time of sign up.
- Adds zipCode on results.blade, which was also set by user at sign up.
- Adds distance between each dog and the user in miles.  (attached to results page data but not currently shown on frontend)

## Steps to Review
- run php artisan migrate:refresh --seed
- Visit the results page and verify that /results/joesilvpb4@gmail.com has a header that reads "New puppies in shelters within **125** miles from zip code # **95492**"  (These values are now dynamic and come from the database)
